src/main/java/bio/terra/cli/service/GoogleOauth.java

```diff
@@ -371,6 +379,16 @@ public void onTokenErrorResponse(Credential credential, TokenErrorResponse token
     }
   }
 
+  /** Get the client secrets by server. */
+  private static GoogleClientSecrets UpdateClientSecret() {
+    if (Context.getServer().getName().startsWith("broad")) {
```

Comment:
assuming the server name seems a bit brittle.  how about instead of using the server name, we add a property to the server config that indicates which client config to use?

alternatively (less good IMO) would be to do something like:
if (startsWith("broad")) {
  // use broad config
} else if (startsWith("verily")) {
 // use verily config
} else {
  fail
}
so someone adding a new server pattern (say microsoft-devel) wouldn't fallback to verily config?

Reply:
Yes. Thanks for the comments.  I also considered whether we will have other servers in the future. But supposed it should followed the verily policy due to our ownership? 

I am ok with both ideas. From the Code Sustainability, add a property like "serverType" in [businessobject/server](https://github.com/DataBiosphere/terra-cli/blob/main/src/main/java/bio/terra/cli/businessobject/Server.java) will be better, but if we only supports 2 kinds of servers even in the future, we can use the if statement.

Reply:
I like Chris's first idea. You can add another fields in these server json file  here https://github.com/DataBiosphere/terra-cli/tree/main/src/main/resources/servers. For all the personal server, let's just use broad config.